.github/workflows/rust.yml

```diff
@@ -24,26 +24,19 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        tool:
-          - name: fmt
-            trailer: --all -- --check
-          - name: clippy --locked
-          - name: clippy --locked
-            trailer: --tests
-          - name: check --locked
-            trailer: --tests
```

Comment:
Likewise the check

Reply:
Similar to above, `--all-targets` implies `--tests`.

Note that a `target` in this context does _not_ refer to what rustc commonly understands as `target`, i.e. the platform to compile for. In Cargo world, a target refers to various _build targets_, such a binary, benchmark, test...